compiler/tflchef/core/src/CustomOp/MaxPoolWithArgMax.cpp

```diff
@@ -29,7 +29,7 @@ MaxPoolWithArgMaxChef::custom_value(flatbuffers::FlatBufferBuilder &fbb) const
 {
   auto &operation = (*_operation);
 
-  assert(operation.type() == "MaxPoolWithArgMax");
+  assert(operation.type() == "MaxPoolWithArgMax" || operation.type() == "MaxPoolWithArgmax");
```

Comment:
Not a typo, I'm guessing?

Reply:
you are right, this the idea of this PR.

Currently we have wrong spelling in both tflchef and recipes.
This PR enables correct behavior and preserves old one.

Why not just rename everything?
Because I can not change tflchef and recipes in one PR and can not disable tests without deletion of recipes.
Separate changes in tflchef or recipes will break dredd tests (and only way to disable them is to delete corresponding recipes, I don't want to do this)

Reply:
I assume you'll remove `MaxPoolWithArgMax` later when all has changed.